proposals/nnnn-android-studio-sdl-plugin.md

```diff
@@ -1,8 +1,8 @@
 # Android Studio IDE SDL Plugins
 
-* Proposal : SDL -nnnn 
+* Proposal : SDL-nnnn 
 * Author : Michael Morgan
-* Status:  Awaiting Review
+* Status:  **Awaiting Review**
 * Review Manager : TDB
 * Impacted Platforms : Android
```

Comment:
Note that these are still incorrect and do not follow the template:

```
* Proposal: [SDL-NNNN](NNNN-filename.md)
* Author: [SDL Developer](https://github.com/smartdevicelink)
* Status: **Awaiting review**
* Impacted Platforms: [Core / iOS / Android / Web / RPC / Protocol]
```

Note the placing of the colons, the capitalization of the N's, the link to the file (just leave it as is in the template, the link to the github account which should link to your github. **Please follow the template.**